tools/format.js

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env -S deno run --unstable --allow-write --allow-read --allow-run
```

Comment:
Unfortunately this doesn't work on linux

Reply:
That's unfortunate :( it does work on `macOS`. I'm not super happy that incantations for linting and formatting will be so long but I think it's worth given we can remove all Python dependencies

Reply:
@ry It does with newer versions of /usr/bin/env. (ie ubuntu 19.04 and up).

Reply:
`env` got -S (to split a single argument string into multiple arguments) in coreutils 8.30.

Some linux distros have 8.29 or lower, you can check [repology for coreutils](https://repology.org/project/coreutils/versions) for a fairly complete list of major linux package repos.